### PR TITLE
fauve-cdb:0.1.0

### PR DESCRIPTION
**Pull request by grodino**
I am submitting
- [x] a new package
- [ ] an update for a package

fauve-cdb:0.1.0 provides a typst template for doctoral dissertations of the French [Collège doctoral de Bretagne
(CdB)](https://www.doctorat-bretagne.fr/). The original LaTeX template can be found
[here](https://gitlab.com/ed-matisse/latex-template).

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/fauve-cdb/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "Timothé Albouy <timothe.albouy@gmail.com>",
+    "Augustin Godinot <https://grodino.github.io>",
+]
+license = "MIT-0"
+description = "The unofficial implementation of the Collège Doctoral de Bretagne thesis manuscript template."
+categories = ["thesis"]
+disciplines = ["computer-science", "mathematics"]
+
+[template]
+path = "example"
```
Comment (laurmaedje) [inline packages/preview/fauve-cdb/0.1.0/typst.toml:15]: It's rather unconventional to use `template` for the package sources and `example` for what would usually be called `template`. Though it's ok if you prefer it like th is.

Comment (grodino) [reply packages/preview/fauve-cdb/0.1.0/typst.toml:15]: Sorry for the delayed answer, I'll change this then.

File: packages/preview/fauve-cdb/0.1.0/template/main.pdf
Comment (laurmaedje) [inline packages/preview/fauve-cdb/0.1.0/template/main.pdf:1]: This file should be deleted.

Comment (grodino) [reply packages/preview/fauve-cdb/0.1.0/template/main.pdf:1]: done !

**Comment by elegaanz** [comment]
Can you please rename the `LICENCE` file to `LICENSE`? After that, it should be good to merge.

**Comment by grodino** [comment]
> Can you please rename the `LICENCE` file to `LICENSE`? After that, it should be good to merge.

Done!

**Comment by elegaanz** [comment]
Thank you :)

**Pull request merged**